addons/core/functions/common/fn_setDefaultLoadouts.sqf

```diff
@@ -0,0 +1,23 @@
+/* ----------------------------------------------------------------------------
+Function: bnb_es_core_fnc_setDefaultLoadouts
+
+Description:
+	Sets the default loadouts for called players.
+
+Parameters:
+	0: _loadouts - The loadouts to include <ARRAY>
+
+Returns:
+	Nothing.
+
+Examples:
+	[_DefaultLoadouts] call bnb_es_core_fnc_setDefaultLoadouts;
+
+Author:
+	Met
+---------------------------------------------------------------------------- */
+
+params [["_loadouts",[]]];
+{
+	[_x select 0, _x select 1] call ace_arsenal_fnc_addDefaultLoadout;
+} forEach _loadouts;
\ No newline at end of file
```

Comment:
Make sure to leave a newline at the end of a file ;)
I don't know what editor you use, but whichever it is I recommend you look into a plugin for it to be able to make use of the `.editorconfig` file, it's what we use to keep consistent code formatting and such.